src/ui/apply.ui

```diff
@@ -1,103 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.38.2 -->
 <interface domain="curtail">
-  <requires lib="gtk+" version="3.24"/>
+  <requires lib="gtk" version="4.0"/>
   <template class="CurtailApplyDialog" parent="GtkDialog">
```

Comment:
This entire widget seems like a usecase for https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.MessageDialog.html